po/ru.po

```diff
@@ -12132,10 +12136,9 @@ msgstr "<b>Описание категории</b>"
 
 #: gnucash/gtkbuilder/assistant-hierarchy.glade:341
 msgid "Choose accounts to create"
-msgstr "Выберите счета для создания"
+msgstr "Выбор создаваемых счетов"
 
```

Comment:
Just a note:
You should in the whole program be consistent to use the polite form of address to sb. or not. I don't know the current usage in Russia.

Reply:
Hello Frank,

the old form was simply translated from English 1:1 and sounded like an instruction - not as a short description of wizard step. I have looked at list with all wizard steps and tried to find a better form: as a short description. 

Thanks, DiMang